src/mach/segment.rs

```diff
@@ -466,6 +466,16 @@ impl<'a> Segment<'a> {
         segment: &SegmentCommand32,
         offset: usize,
         ctx: container::Ctx,
+    ) -> Result<Self, error::Error> {
+        Self::from_32_2(bytes, segment, offset, ctx, false)
+    }
+
+    pub(crate) fn from_32_2(
```

Comment:
i'd rename this `from_32_impl` or something, change it to private, and have `from_32` call it with the boolean flag = false, and `from_32_lossy` call it with true